build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.10.4-1
+v1.11.1-1
```

Comment:
I've built and pushed k8s.gcr.io/kube-cross:v1.11.1-1. Waiting for it to sync to mirrors.